code/game/world.dm

```diff
@@ -3,6 +3,10 @@
 GLOBAL_LIST_INIT(map_transition_config, MAP_TRANSITION_CONFIG)
 
 /world/New()
+	//temporary file used to record errors with loading config, moved to log directory once logging is set up
+	GLOB.config_error_log = GLOB.world_game_log = GLOB.world_runtime_log = "data/logs/config_error.log"
+	load_configuration()
+	// Setup all log paths and stamp them with startups
 	SetupLogs()
```

Comment:
The issue last time was I had this **after** `SetupLogs()` when it should have been before.